app/src/main/java/eu/kanade/tachiyomi/data/database/DbOpenCallback.kt

```diff
@@ -20,11 +20,12 @@ class DbOpenCallback : SupportSQLiteOpenHelper.Callback(DATABASE_VERSION) {
         /**
          * Version of the database.
          */
-        const val DATABASE_VERSION = 13
+        const val DATABASE_VERSION = 14
     }
 
     override fun onCreate(db: SupportSQLiteDatabase) = with(db) {
         execSQL(MangaTable.createTableQuery)
+        execSQL(MangaTable.addFilteredScanlators)
```

Comment:
I couldn't get it to add this col for some reason without this... so not sure if this should be here

Reply:
it shouldn't, it might be that you updated the db before adding the version check, but removing this should be fine now for upgrading users

Reply:
not sure how to delete the db because i've been doing clear app data and uninstalling but i still need this line >:c so I guess when testing I'll just have to add that back in. I'm under the impression createTableQuery should make it normally